docs/application/dotnet/guides/hot-reloading/overview.md

```diff
@@ -0,0 +1,71 @@
+# Hot Reloading
+
+Hot reloading is to inject a new files being edited without stopping the running app. In this way, you don't lose state of the app that is especially useful when editing UI.
+
+The following video demonstrates hot reloading an app running on Tizen TV emulator.
+
+![test_video](./media/hotreloading1.webp)
+
+## Beholder
+
+Beholder is a hot reloading support tool for programs using Tizen.NET's NUI.
+
+Hot reloading refreshes the files that were changed, without losing the state of the app.
+
+After launching the application on the emulator or real device, you can apply the changed file without stopping the application.
+
+If in the screen there are several stages, you can update the screen without taking a step back, changes are applied to all files.
+
+
+## Enable hot reloading
+
+### Install beholder with private feed
+
+Add following feed to your VS extension private gallery.
+* https://nui-dali.github.io/Beholder/gallery/feed.xml
+
+![extension_setup](./media/extension_setup.png)
+
+### Add a private gallery to extensions and updates
+
+1. On the menu bar, select **Tools > Options**.
+2. In the **Environment** node, select **Extensions**.
+3. Select the **Add** button.
+4. In the **Name** field, enter a name for the beholder.
+5. In the **URL** field, enter the URL.
+
+### Install extensions from a private gallery
+
+To search for and install extensions from a private gallery, perform following steps:
+
+1. On the menu bar, select **Extensions > Manage Extensions**.
+2. In the left pane, select **Online Extensions**, and then select **Beholder**.
+3. In the right pane, select **Tizen.NUI.Hotreloading**, and then select the **Download** button.
+
+You can learn more about how to install your personal gallery in [this MS article](https://docs.microsoft.com/en-us/visualstudio/extensibility/private-galleries?view=vs-2019).
+
+![extension_setup](./media/extension_install.png)
+
+### Beholder Usage
+
+1. Open your NUI project.
+2. On the menu bar, select  **Tools > Enable Beholder**.
+3. On the menu bar, select  **Debug > Run without Debug**.
+4. Open and edit the xaml file.
+5. Save the xaml file.
+
+![Beholder_Menu](./media/menu.png)
```

Comment:
It seems the image is captured after above process. (The menu shows Disable Beholder.)
How about using an image captured during the process?